k2mosaic/mosaic.py

```diff
@@ -183,6 +200,8 @@ def _make_primary_hdu(self):
         hdu.header.cards['TELESCOP'].comment = 'telescope'
         hdu.header['INSTRUME'] = 'Kepler Photometer'
         hdu.header.cards['INSTRUME'].comment = 'detector type'
+        hdu.header['FILTER'] = 'Kepler'
+        hdu.header.cards['FILTER'].comment = 'filter'
```

Comment:
I guess we can be a bit more informative in this comment, I suggest:
```suggestion
        hdu.header.cards['FILTER'].comment = 'photometric passband'
```